tests/e2e-leg-6/collect-scrutinize/25-check-tarball.yaml

```diff
@@ -14,4 +14,6 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - command: kubectl exec vertica-scrutinize-vdb-exist -n $NAMESPACE -c main -- sh -c "ls | grep -q VerticaScrutinize.*.tar"
\ No newline at end of file
+  - script: |-
+      tarball=$(kubectl -n $NAMESPACE get vscr vertica-scrutinize-vdb-exist -o json | jq '.status.tarballName' | grep -Eq '^VerticaScrutinize\.[0-9]{14}$')
+      kubectl exec vertica-scrutinize-vdb-exist -n $NAMESPACE -c main -- ls | grep -q $tarball\.tar
```

Comment:
I kind of expected that the file name we put in the status is the full file name. Do you think it's a bit odd that we have to append .tar to get the actual file name? It's not always convention that a tarball ends with .tar. If it's also gziped they end with .tar.gz.

Reply:
I did not want any dependencies on the extension. We can append .tar assuming we will stick with that in vcluster.

Reply:
Yeah, maybe we just need to append '.tar' in `getTarballName`